src/sagas/lists.js

```diff
@@ -1,7 +1,7 @@
 import { call, put, select, takeEvery, takeLatest } from 'redux-saga/effects';
 import { filtersTransformer } from '../filters';
 import Api from '../helpers/Api';
-import { COLLECTION_LESSONS_TYPE, CT_VIDEO_PROGRAM_CHAPTER, UNIT_LESSONS_TYPE } from '../helpers/consts';
```

Comment:
Why you remove UNIT_LESSONS_TYPE?

Reply:
I rename it from lesson to section, it can be one of sections (lesson, events...)